packages/yarnpkg-pnpify/sources/commands/SdkCommand.ts

```diff
@@ -13,24 +13,27 @@ export default class SdkCommand extends Command {
   @Command.String(`--cwd`)
   cwd: NativePath = process.cwd();
 
+  @Command.Boolean(`-v,--verbose`)
+  verbose: boolean = false;
+
   static usage = Command.Usage({
     description: `generate editor SDKs and settings`,
     details: `
-      This command generates a new directory, \`.yarn/pnpify\`, which includes the base SDKs.
+      This command generates a new directory, \`.yarn/sdks\`, which includes the base SDKs.
 
       When used without arguments, it:
 
       - throws an error on non-pnpified projects
 
       - updates all existing SDKs and editor settings on already-pnpified projects
 
-      The optional \`integrations\` rest argument is a list of supported integrations or \`base\`.
+      The optional integrations arguments are a set of supported integrations, or the keyword \`base\`.
 
-      - When \`base\`, it only installs the base SDKs. Useful for when an editor is not yet supported and you need to manually update its settings.
+      - When \`base\` is used, only the base SDKs will be generated. This is useful for when an editor is not yet supported and you plan to manage the settings yourself.
 
-      - When a list of editors (e.g. \`vscode vim\`), it installs the base SDKs and generates the corresponding editor settings.
+      - When a set of integration is used (e.g. \`vscode\`, \`vim\`, ...), the base SDKs will be installed plus all the settings relevant to the corresponding environments (for example on VSCode it would set \`typescript.tsdk\`).
```

Comment:
```suggestion
      - When a set of integrations is used (e.g. \`vscode\`, \`vim\`, ...), the base SDKs will be installed plus all the settings relevant to the corresponding environments (for example on VSCode it would set \`typescript.tsdk\`).
```